sky/core.py

```diff
@@ -551,10 +555,13 @@ def cancel(cluster_name: str,
         cluster_name, operation_str='Cancelling jobs')
 
     # Check the status of the cluster.
-    handle = backend_utils.check_cluster_available(
-        cluster_name,
-        operation='cancelling jobs',
-    )
+    if not _ignore_cluster_aliveness:
+        handle = backend_utils.check_cluster_available(
```

Comment:
Can we document the diff between check_cluster_available vs. refresh_cluster_status_handle? The Perhaps add it to the former's docstr?

Reply:
I re-implemented this part and added a comment below. PTAL.